src/main/java/com/algorand/algosdk/crypto/Address.java

```diff
@@ -110,6 +128,38 @@ public String encodeAsString() throws NoSuchAlgorithmException {
         return res;
     }
 
+    /**
+     * verifyBytes verifies that the signature for the message is valid for the public key
+     * @param message the message that was signed
+     * @param signature
+     * @return boolean; true if the signature is valid
+     */
+    public boolean verifyBytes(byte[] message, byte[] signature) throws NoSuchAlgorithmException, InvalidKeySpecException, InvalidKeyException, SignatureException {
```

Comment:
A second benefit of wrapping `byte[] signature` in a `Signature` is you get length checking for free